wagtail/admin/urls/pages.py

```diff
@@ -87,21 +86,6 @@
         workflow.PreviewRevisionForTask.as_view(),
         name="workflow_preview",
     ),
-    path(
-        "moderation/<int:revision_id>/approve/",
-        moderation.approve_moderation,
-        name="approve_moderation",
-    ),
-    path(
-        "moderation/<int:revision_id>/reject/",
-        moderation.reject_moderation,
-        name="reject_moderation",
-    ),
-    path(
-        "moderation/<int:revision_id>/preview/",
-        moderation.preview_for_moderation,
-        name="preview_for_moderation",
-    ),
```

Comment:
There are references to this URL in `wagtailadmin/notifications/submitted{_subject.txt,.txt,.html}` templates, and I think we can also remove `send_moderation_notification` in `wagtail/admin/mail.py`. I believe they have been replaced by `WorkflowStateSubmissionEmailNotifier` and the `workflow_state_submitted{_subject.txt,.txt,.html}` templates.

There might be tests that refer to the old notification system. I completely missed this part!